tools/quicktype-wrapper/src/index.ts

```diff
@@ -67,7 +67,7 @@ async function getJSONSchemasPaths(directory: string) {
  * @returns {string} The filename, like document_event_data
  */
 function getFilename(typeName: string, lang: string) {
-  if (lang === 'GOLANG') {
+  if (lang === 'GOLANG' || lang === 'RUBY') {
```

Comment:
A comment along the lines of "Golang and Ruby prefer filenames to use lower snakecase instead of CamelCase" would be helpful...

Reply:
Added.